### DISCUSSION TITLE
C Make IBAMR, not locating NumDiff

### DISCUSSION THREAD

**Discussion by agiordan96** [IBAMR/IBAMR#1713] [Q&A] [answered]
Hello everyone, 

I am aware my question is similar to that of @amneetb, however it is not clear how it was handled. While configuring IBAMR with CMake, NumDiff root is not located. 

`CMake Warning at tests/CMakeLists.txt:319 (MESSAGE):
  The configuration script was not able to locate numdiff.  If you want to
  run the test suite you will need to either edit attest.conf, specify the
  path to numdiff to attest, or rerun CMake with the argument NUMDIFF_ROOT
  specifying numdiff's root installation directory.`
  
  Which one of the several NumDifference.h should I pass? There are several, in lib mesh there is one, in ibtk there is one, in opt/homebrew/opt/eigen another one...
  It appears no one satisfies this request, as the above message persists. Could you please help?
[CMakeCache.txt](https://github.com/IBAMR/IBAMR/files/15470953/CMakeCache.txt)

**Comment by boyceg** [comment, marked as answer]
`numdiff` is a program. On MacOS I would recommend installing it with Homebrew, use `brew install numdiff`. Then the location should be something like `/opt/homebrew/bin/numdiff`.
